include/kernel/thread.h

```diff
@@ -307,5 +309,21 @@ void awaken(void);
 /* Place the thread bound to this scheduling context in the release queue
  * of periodic threads waiting for budget recharge */
 void postpone(sched_context_t *sc);
+
+static inline ticks_t getNextInterrupt(void)
```

Comment:
The name 'getNextInterrupt' is a bit misleading. It's the remaining time slice?

Reply:
is this not the existing name?